server/base/src/main/java/org/apache/accumulo/server/security/SecurityUtil.java

```diff
@@ -49,11 +52,10 @@ public static void serverLogin(AccumuloConfiguration acuConf) {
 
     if (login(principalConfig, keyTab)) {
       try {
-        // This spawns a thread to periodically renew the logged in (accumulo) user
-        UserGroupInformation.getLoginUser();
+        startTicketRenewalThread(UserGroupInformation.getCurrentUser(), acuConf.getTimeInMillis(Property.GENERAL_KERBEROS_RENEWAL_PERIOD));
```

Comment:
> Do we want to start with UserGroupInformation.getCurrentUser

I'm not sure what you mean by "start with". It's just a getter -- there is no renewal for keytab-based logins.